### PR TITLE
v0.3.0

### PR DESCRIPTION
**Pull request by CagtayFabry**
I think this should be ready now

## changes
- update CHANGELOG.md for v0.3.0 release
- add AWS shielding gas schemas to documentation

### CODE REVIEW

**Review by marscher — APPROVED** [review]
LGTM! Great job :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release Notes
 
-## 0.3.0 (unreleased)
+## 0.3.0 (11.03.2021)
```
Comment (vhirtham) [inline CHANGELOG.md:3]:
```suggestion
## 0.3.0 (12.03.2021)
```
😛

**Review by vhirtham — APPROVED** [review]

**Comment by vhirtham** [comment]
The bugfix in the other PR should be included in the release

**Comment by CagtayFabry** [comment]
will merge after #275 and #276

File: CHANGELOG.md (hunk 1 above)
Comment (CagtayFabry) [reply CHANGELOG.md:3]: that was what I originally wanted to put there lol

**Pull request merged**